packages/liquidator/index.js

```diff
@@ -46,7 +48,9 @@ async function run({
   errorRetriesTimeout,
   priceFeedConfig,
   liquidatorConfig,
-  liquidatorOverridePrice
+  liquidatorOverridePrice,
+  startingBlock,
```

Comment:
as discussed on slack, you could consider putting these within the `liquidatorConfig`.

Reply:
I'm going to pack `startingBlock` and `endingBlock` into `liquidatorConfig` before creating the `Liquidator.js` object, but I still want to read those vars from the environment because the serverless hub sets them conveniently there

Reply:
See [this commit](https://github.com/UMAprotocol/protocol/pull/2432/commits/2717599ae0fc024fd0437a07c8f4712dc81f445c) for the proposed changes in response to your comment.